setup.py

```diff
@@ -1,11 +1,27 @@
 #!/usr/bin/env python
 
+VERSION = '1.0.0'
+
 import glob
 from setuptools import setup
 setup(
     name = "hocr_tools",
-    version = "0.1",
-    author = 'Thomas Breuel',
+    version = VERSION,
     description = 'Advanced tools for hOCR integration',
+    author = 'Thomas Breuel',
+    author_email = 'tmbdev@gmail.com',
+    url = 'https://github.com/tmbdev/hocr-tools',
+    download_url = 'https://github.com/tmbdev/hocr-tools/tarball/' + VERSION,
```

Comment:
I think we need here the `v` as well, e.g. something like
```
https://github.com/zuphilip/hocr-tools/tarball/v0.3.2
```
which just redirects to
```
https://codeload.github.com/zuphilip/hocr-tools/legacy.tar.gz/v0.3.2
```